ADAL/src/cache/ADTokenCacheAccessor.m

```diff
@@ -116,16 +130,18 @@ - (ADTokenCacheItem *)getMRRTItemForUser:(ADUserIdentifier *)identifier
 - (ADTokenCacheItem *)getFRTItemForUser:(ADUserIdentifier *)identifier
                                familyId:(NSString *)familyId
                           correlationId:(NSUUID *)correlationId
+                     telemetryRequestId:(NSString*)telemetryRequestId
                                   error:(ADAuthenticationError * __autoreleasing *)error
 {
     NSString* fociClientId = [ADTokenCacheAccessor familyClientId:familyId];
     
-    return [self getItemForUser:identifier resource:nil clientId:fociClientId correlationId:correlationId error:error];
+    return [self getItemForUser:identifier resource:nil clientId:fociClientId correlationId:correlationId telemetryRequestId:telemetryRequestId error:error];
 }
 
 - (ADTokenCacheItem*)getADFSUserTokenForResource:(NSString *)resource
                                         clientId:(NSString *)clientId
                                    correlationId:(NSUUID *)correlationId
```

Comment:
Passing in both a correlationId and a telemetryRequestID is getting to be a bit much, we should probably have a per-request context object that we pass in instead that contains both of these.